src/lib/questions-utils.property.spec.ts

```
@@ -0,0 +1,185 @@
+import { describe, it, expect, vi } from "vitest";
+import fc from "fast-check";
+import { reorderQuestionsByIds } from "./questions-utils.ts";
+
+type Question = {
```

ESLint

[error] Use an `interface` instead of a `type`.

(@typescript-eslint/consistent-type-definitions)